resources/check.rb

```diff
@@ -30,9 +30,9 @@
 provides :sensu_check
 
 property :check_hooks, Array
-property :command, String, default: '/bin/true', required: true
+property :command, String, default: '/bin/true'
 property :cron, String
-property :handlers, Array, default: [], required: true
+property :handlers, Array, default: []
```

Comment:
is there a handle false key like there was previously? At my org we always required a handler to be specified even if you disable handling. This makes the most sense to me of the ones to remove the required input.

Reply:
This is also because of the cookstyle: https://www.rubydoc.info/gems/cookstyle/RuboCop/Cop/Chef/ChefRedundantCode/PropertyWithRequiredAndDefault

Sensu Go doesn't appear to have a handle false key like Sensu Core, primarily because there's no default handler. If you want to have an alert with a critical threshold, but don't want to be alerted, in Sensu Go you just don't add a handler to the check. Keeping this with a default of an empty array would keep people from needing to specify the empty array for their checks.

Reply:
it does look like its not required according to the spec while the other are.